htdocs/admin/index.php

```diff
@@ -66,6 +68,7 @@
           <td>
             <a href="edit_challenge.php?id=', htmlspecialchars($challenge['id']), '" class="btn btn-xs btn-primary">Edit</a>
             <a href="new_hint.php?id=', htmlspecialchars($challenge['id']), '" class="btn btn-xs btn-warning">Hint</a>
+            <img src="/img/', ((int)$challenge['expose'] ==1 ? 'accept.png" alt="Exposed"' : 'stop.png" alt="Not Exposed"'), ' />
```

Comment:
Our environment doesn't allow for comparing integer results as boolean. When we tried doing the example you suggested in our setup the result comes back as 0 or 1 and comparing as boolean doesn't work. That's also the reason I removed all those other checks in PHP and just added it to the SQL query.